core/src/main/scala/tech/beshu/ror/accesscontrol/factory/decoders/AuditingSettingsDecoder.scala

```diff
@@ -182,6 +210,12 @@ object AuditingSettingsDecoder extends Logging {
       .overrideDefaultErrorWith(error)
   }
 
+  private def unsupportedOutputTypeError(unsupportedType: String) = {
+    AuditingSettingsCreationError(Message(
+      s"Unsupported 'type' of audit output: $unsupportedType. Supported types: [index, log]"
```

Comment:
let's pass the supported types list in the param. If some strings are repeated, it's better to have them closed each other (now it's high probability that someone in the future forgets to add the new supported type here)